src/dji_osdk_ros/modules/dji_vehicle_node_publisher.cpp

```diff
@@ -910,4 +910,17 @@ void VehicleNode::publishCameraH264(uint8_t* buf, int bufLen, void* userData)
   }
 }
 
+void VehicleNode::streamCamera(CameraRGBImage rgbImg, void* userData)
+{
+  if (userData)
+  {
+    VehicleNode *node_ptr = reinterpret_cast<VehicleNode*>(userData);
```

Comment:
since userData is void* it is unspecified. It is recommended to use `static_cast()` instead. Look into https://stackoverflow.com/questions/573294/when-to-use-reinterpret-cast

Reply:
Resolved in [720d4e4](https://github.com/umdlife/Onboard-SDK-ROS/pull/4/commits/720d4e47b2cc6c198611cb54eceacb90b40fa57d)